corehq/apps/users/models.py

```diff
@@ -163,6 +163,7 @@ def allowed_items(self):
     'view_tableau': 'view_tableau_list',
     'access_web_apps': 'web_apps_list',
     'commcare_analytics_roles': 'commcare_analytics_roles_list',
+    'edit_profiles_list': 'edit_profiles_list',
```

Comment:
I think the key here should be `edit_all_user_profiles`. I'm looking at how `view_tableau` and `access_web_apps` are done and how it is used in `roles.js`